ocaml/utils/language_extension_kernel_intf.mli

```diff
@@ -0,0 +1,71 @@
+(** These module types make it easier to import more source files directly
+    from the compiler into the [ppxlib_jane] open-source library. We copy source
+    files because the [ppxlib_jane] library can't rely on Jane Street's OCaml
+    compiler libraries -- it must remain compatible with the upstream OCaml
+    compiler.
```

Comment:
This isn't true anymore; does that change the design here?

Reply:
Let's chat tomorrow. This is still true. The relevant thing to think about is upstream versions of public-release ppxes.